test/WebJobs.Script.Tests.Integration/Management/FunctionsSyncManagerTests.cs

```diff
@@ -311,6 +311,21 @@ public async Task TrySyncTriggers_LocalEnvironment_ReturnsFalse()
             }
         }
 
+        [Fact]
+        public async Task TrySyncTriggers_ManagedAppEnv_WithNo_AzureWebJobsStorage_ReturnsTrue()
```

Comment:
Can you add a test for the existing Kubernetes Managed Hosting case as well? That way we have test coverage for all these Kubernetes scenarios.